authapi/permissions.py

```diff
@@ -237,10 +241,14 @@ def has_object_permission(self, request, view, obj):
 
     def user_has_permission(self, user, permission_type, object_id=None):
         permissions = get_user_permissions(user)
-        permissions = find_permission(permissions, permission_type, object_id)
+        permissions = find_permission(
+            permissions, permission_type, object_id,
+            settings.PERMISSION_NAMESPACE)
         return permissions.exists()
 
-    def check_permissions(self, user, ptype, object_id):
+    def check_permissions(self, user, ptype, object_id, namespace):
+        if namespace != settings.PERMISSION_NAMESPACE:
+            return True
```

Comment:
I think that the `test_permission_add_permission_read_access` does test this. It tests that a user without any permissions can add a permission with namespace `foo`. Do you mean something more explicit?